app/src/test/java/com/futurice/freesound/feature/search/DefaultSearchDataModelTest.java

```diff
@@ -71,20 +70,20 @@ public void querySearch_emitsSingleUnit_whenQuerySearchSuccessful() {
         new Arrangement()
                 .withSearchResultsFor(QUERY, dummyResults());
 
-        TestSubscriber<Void> ts = defaultSearchDataModel.querySearch(QUERY).test();
-
-        ts.assertNoErrors()
-          .assertComplete();
+        defaultSearchDataModel.querySearch(QUERY)
+                              .test()
+                              .assertNoErrors()
```

Comment:
This one is a completable, you already assert it to be completed. In that case it won't have an error